### ISSUE TITLE
StreamJsonRpc.RemoteInvocationException opening VSMEF solution

### ISSUE THREAD

**Issue by KevinH-MS** [dotnet/roslyn#16401]
This is a "broken" solution, but I need VS not to crash so I can fix it... 😄 

1.  Open solution at [well known internal share]\kevinh\VSMEF
2.  Open a file in some project
3.  Wait

**RESULTS:**
VS Crash
```
StreamJsonRpc.RemoteInvocationException:
Unexpected false
   at Roslyn.Utilities.Contract.ThrowIfFalse(Boolean condition, String message)
   at Microsoft.CodeAnalysis.Remote.SolutionCreator.<CreateSolutionAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at Microsoft.CodeAnalysis.Remote.SolutionService.<UpdatePrimaryWorkspaceAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.CodeAnalysis.Remote.RemoteHostService.<SynchronizePrimaryWorkspaceAsync>d__5.MoveNext()
```

**Comment by KevinH-MS** [comment]
FYI @heejaechang

**Comment by KevinH-MS** [comment]
I'm seeing this on the latest d15rel bits.

**Comment by heejaechang** [comment]
@KevinH-MS let me take a look now, for now, you can turn off OOP through reg key and keep working on the solution.

**Comment by heejaechang** [comment]
@KevinH-MS It doesn't crash on me. let me know when you are avail.

**Comment by srivatsn** [comment]
Fixed by #16421
